client/src/Components/Home/Members/index.jsx

```diff
@@ -1,5 +1,68 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Row, Col } from 'react-bootstrap';
 
-export default function Members() {
-  return <h1> members cards</h1>;
+import './style.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Members({ filtered, history }) {
+  return (
+    <Row key={Math.random()}>
```

Comment:
@Fatmasiam You might be able to remove this random() too, if there is only one of this component being created inside its parent (and not a list of them).  If react throws an error when you remove the key, you could try using `filtered && filtered[0].id` as the key.